third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -369,60 +367,23 @@ function Target_topFrameLocation() {
   return { location: createProtocolLocation(location)[0] };
 }
 
-// Get the raw call frames on the stack, eliding ones in scripts we are ignoring.
+/**
+ * Get the raw call frames on the stack, eliding ones in scripts we are ignoring.
+ * @return {{ callFrames: CDP.Debugger.CallFrame[] }}
+ * 
+ * @see https://chromedevtools.github.io/devtools-protocol/v8/Debugger/#type-CallFrame
+ * @see https://github.com/replayio/chromium-v8/blob/37d50784b68747e7b2d5ebc16305cb9b3227741a/src/inspector/v8-debugger-agent-impl.cc#L1412
+ */
 function getStackFrames() {
+  // NOTE: this is a custom command we added
   const { callFrames } = sendMessage("Debugger.getCallFrames");
   return callFrames;
 }
 
 
-window.DevOnly = {
-  // NOTE: commented out until we have a safe/reliably way of dealing with it
-  // tryEvalDev(expression, frameId = 0) {
-  //   // log(`[CHROMDEBUG] eval - expression: "${expression}"`);
-
-  //   // NOTE: expression sometimes gets wrapped in parentheses, and its value must be a string
-  //   const prefixes = ['("dev:', '"dev:'];
-  //   const prefix = prefixes.find(p => expression.startsWith(p));
-  //   if (prefix) {
-  //     // hackfix: evaluate straight-up in our dev context
-  //     // TODO: unsafe. Must be behind DEV-ONLY flag.
-
-  //     let cmd = expression;
-  //     if (cmd.startsWith('(')) {
-  //       // strip '()'
-  //       cmd = cmd.substring(1, expression.length - 1);
-  //     }
-  //     if (cmd.endsWith(';')) {
-  //       // strip trailing ';'
-  //       cmd = cmd.substring(0, expression.length - 1);
-  //     }
-
-  //     // parse JSON (used for serialization)
-  //     cmd = JSON.parse(cmd);
-
-  //     // strip "dev:" and wrap in ()
-  //     cmd = `(${cmd.substring(4)})`;
-
-  //     // run
-  //     const res = eval(cmd);
-  //     const resJson = JSON.stringify(res);
-
-  //     const t = res !== undefined ? ` (type: ${typeof res})` : '';
-  //     // log(`[CHROMDEBUG] eval (dev) - cmd: "${cmd}", res:${t} "${resJson}"`);
-
-  //     return { result: { data: {}, returned: { value: resJson } } };
-  //   }
-  // },
-
-  // popCommandError() {
-  //   return gLastCommandErrors.shift();
-  // }
-};
-
 // Build a protocol Result object from a result/exceptionDetails CDP rval.
-function buildProtocolResult({ result, exceptionDetails }) {
-  const value = remoteObjectToProtocolValue(result);
+function buildRrpObjectResult({ result, exceptionDetails }) {
```

Comment:
What's rrp in this context?

Reply:
CDP = Chrome Debugger Protocol
vs.
RRP = Record Replay Protocol